llvm/lib/DebugInfo/Symbolize/MarkupFilter.cpp

```diff
@@ -698,6 +695,24 @@ void MarkupFilter::reportLocation(StringRef::iterator Loc) const {
   errs() << '\n';
 }
 
+void MarkupFilter::symbolizeFailure(const MarkupNode &Node, const Module *Mod,
+                                    Error SymbolizeError) {
+  if (Mod) {
```

Comment:
In the three call sites you've modified, the `Mod` must always be present, so this feels like an unnecessary check?